bftclient/src/matcher.h

```diff
@@ -26,6 +26,8 @@ struct MatchConfig {
   // All quorums can be distilled into an MofN quorum.
   MofN quorum;
   uint64_t sequence_number;
+  bool ignore_primary_ =
```

Comment:
I think negative booleans are hard to reason about. Would you mind changing this to `include_primary_` and defaulting it to true. Otherwise it reads like 'don't ignore primary' which is kind of a confusing statement and less straightforward than something like 'don't include primary'.

Reply:
fixed